carpics_360v0.1.js

```diff
@@ -80,10 +81,12 @@ var CarPicsSpinnerAPI = (function() {
         this.AllLoadedFunctions = [];
         this.turnStatus = false;
         this.turning = false;
+        this.multiZoom=true;
         this.lastTouch = 0;
         this.autospinSleep = config.autospinSleep > 0 ? config.autospinSleep : 100;
         this.mouseDisabled = config.disableMouse || false;
         this.zoomed = false;
+        this.displayHotspots = true;
```

Comment:
For this and the new inertial moving behaviours we will include some additional configurations.  Ask me about this tomorrow.